vb-fe/src/_store/room/RoomActions.ts

```diff
@@ -12,8 +13,35 @@ export const createRoomAction =
 
     dispatch({
       type: SystemConstants.LOADING,
-      payload: true,
+      payload: false,
     });
-    const user: any = JSON.parse(localStorage.getItem("v-user") || "");
-    socket?.emit("create-room", { room, userId: user["_id"] }); //eslint-disable-line
+
+    try {
+      const user: ILocalStorageUser = JSON.parse(localStorage.getItem("v-user") || "");
```

Comment:
JSON parsing an empty string just throws an error.  Although we can't guarantee the type, I thought this would make organization better, since we know what we're expecting